src/pkg/blob/dao/dao.go

```diff
@@ -222,10 +222,6 @@ func (d *dao) ListBlobs(ctx context.Context, query *q.Query) ([]*models.Blob, er
 	if err != nil {
 		return nil, err
 	}
-
-	if query.Sorting != "" {
-		qs = qs.OrderBy(query.Sorting)
-	}
 	blobs := []*models.Blob{}
```

Comment:
The following in gc job should change to `Sorts` if `query.Sorting !=""` in this method was removed.
```golang
			blobRG := q.Range{
				Min: lastBlobID,
			}
			q := &q.Query{
				Keywords: map[string]interface{}{
					"update_time": &timeRG,
					"projectID":   p.ProjectID,
					"id":          &blobRG,
				},
				PageNumber: 1,
				PageSize:   int64(ps),
				Sorting:    "id",
			}
			blobs, err := gc.blobMgr.List(ctx.SystemContext(), q)
```

Reply:
done